### PR TITLE
Initial commit for migrating the code from wanghaoran1988 to openshift.

### PR DESCRIPTION
**Pull request by wanghaoran1988**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,63 @@
+FLAGS =
+TESTENVVAR =
+REGISTRY = quay.io/redhat
```
Comment (aditya-konarde) [inline Makefile]: Can we already push to this registry, or is this a placeholder?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,8 @@
+FROM alpine:3.7
```
Comment (aditya-konarde) [inline Dockerfile]: Just something we should do later: https://github.com/openshift/openshift-state-metrics/issues/3

File: README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+[![Build Status](https://travis-ci.org/openshift/openshift-state-metrics.svg?branch=master)](https://travis-ci.org/wanghaoran1988/openshift-state-metrics)
```
Comment (aditya-konarde) [inline README.md]: @wanghaoran1988 link needs fixing: https://travis-ci.org/wanghaoran1988/openshift-state-metrics

Will this be fixed once we get travis running for the new repo?

File: Makefile (hunk 1 above)
Comment (wanghaoran1988) [reply Makefile]: We already can do this, I have the permission now.

File: manifests/openshift-state-metrics-cluster-role.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+apiVersion: rbac.authorization.k8s.io/v1
```
Comment (aditya-konarde) [inline manifests/openshift-state-metrics-cluster-role.yaml]: I really like how you have added only the necessary permissions in the Clusterrole :)

File: manifests/openshift-state-metrics-deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+      serviceAccountName: openshift-state-metrics
+      containers:
+      - name: openshift-state-metrics
+        image: quay.io/redhat/openshift-state-metrics
+        ports:
+        - name: http-metrics
+          containerPort: 8080
+        - name: telemetry
+          containerPort: 8081
+        readinessProbe:
```
Comment (aditya-konarde) [inline manifests/openshift-state-metrics-deployment.yaml]: https://github.com/openshift/openshift-state-metrics/issues/5

File: README.md (hunk 3 above)
Comment (wanghaoran1988) [reply README.md]: yes

**Pull request merged**